src/Advection/upwind_biased_reconstruction.jl

```diff
@@ -23,7 +23,7 @@ struct UpwindBiased{N, FT, XT, YT, ZT, CA, SI} <: AbstractUpwindBiasedAdvectionS
     
     "reconstruction scheme used near boundaries"
     boundary_scheme :: CA
-    "reconstruction scheme used for symmetric interpolation, capped at Centered(order = 4) for stability purposes"
+    "reconstruction scheme used for symmetric interpolation"
```

Comment:
Why do we use lowercase without punctuation for these comments? @navidcy is that recommended?

Reply:
not sure what's the recommended style...

the punctuation was dropped here because I thought it wasn't necessary for this particular comment